libsplinter/src/node_registry/mod.rs

```diff
@@ -79,23 +79,26 @@ impl MetadataPredicate {
 
 /// Provides Node Registry read capabilities.
 pub trait NodeRegistryReader: Send + Sync {
-    /// Returns a list of nodes.
+    /// Returns an iterator over the nodes in the registry.
```

Comment:
Squash typo: fixup_iter after the signature should be removed.

Reply:
Fixed.